Biz/Model/UserPasswordForm.pm

```diff
@@ -18,6 +18,7 @@ sub execute_ok {
         realm_owner => $req->get_nested(qw(auth_realm owner))
             ->update_password($self->get('new_password')),
     });
+    $self->new_other('LoginAttempt')->create({login_attempt_state => $_LAS->SUCCESS});
```

Comment:
Do not do this. This is not a login attempt.

Reply:
FWIW, this was here to allow for the account to be unlocked via the reset password email. If support has to unlock the account, then yes this can be removed.